twisted/test/test_defer.py

```diff
@@ -2446,3 +2446,23 @@ def test_cancelDeferUntilLockedWithTimeout(self):
         self.assertFalse(timeoutCall.active())
         self.assertIsNone(self.lock._timeoutCall)
         self.failureResultOf(deferred, defer.CancelledError)
+
+
+
+class DeprecationTests(unittest.TestCase):
```

Comment:
minor comment... I think that this should be named `TimeoutErrorTests` as we should group multiple unit tests around a single class.